### PR TITLE
docs(api): API definition

### PR DESCRIPTION
**Pull request by jorge-sanz**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: API.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+      "name": "Salad",
+      "description": "Tomatoes, lettice and bacon."
+      "image": "https://www.google.es/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwjknLawgs3TAhULVxQKHfbcAvMQjRwIBw&url=http%3A%2F%2Fjetspizza.com%2Fmenu%2Fsalads&psig=AFQjCNE447cnpJfGRhiwiN5Hys_XLrgpmQ&ust=1493670502918204"
+    },
+    ...
+  ]
+}
+```
+
+## Menu choice
```
Comment (josedl4) [inline API.md:304]: If you add this method here, why you don't  add in book the method of lending a book. I think that is necessary create in book this method.

File: API.md [hunk 2]
```diff
@@ -0,0 +1,664 @@
...
+    "date": "2017-04-04 18:05",
+    "patient": { ... },
+    "type": "...",
+    "completed": true,
+    "description": "Description: ..." 
+  },
+  ...
+]
+```
+
```
Comment (jorge-sanz) [inline API.md:482]: Getting orders from a specified user endpoint for the Patients app is missing.

File: API.md [hunk 3]
```diff
@@ -0,0 +1,664 @@
...
+  "patient": "john",
+  "firstCourse": 33,
+  "secondCourse": 43,
+  "dessert": 53
+}
+```
+
+
+# Items
+## Add item
```
Comment (jorge-sanz) [inline API.md]: For the Shop Epic: add, remove and edit items are not required methods. We are not going to implement that part in workers app.

File: API.md [hunk 4]
```diff
@@ -0,0 +1,664 @@
...
+    "id": 12,
+    "name": "The Times",
+    "price": 1.5,
+    "image": "https://upload.wikimedia.org/wikipedia/en/3/3e/The-Times-5-June-2013.jpg",
+    "description": "the description of this..."
+  },
+  ...
+]
+```
+
```
Comment (jorge-sanz) [inline API.md]: Order a product on shop is missing.

**Review by josedl4 — APPROVED** [review]

**Pull request merged**